futurenet/supervisor/etc/supervisord.conf.d/soroban-rpc.conf

```diff
@@ -3,6 +3,7 @@ user=stellar
 directory=/opt/stellar/soroban-rpc
 command=/opt/stellar/soroban-rpc/bin/start
 autostart=true
+startretries=50
```

Comment:
this enables the soroban rpc service to be put through a incremntal backoff retry loop by supervisor, is needed as soroban rpc will exit at startup time if at least one archive is not available from the `HISTORY_ARCHIVE_URLS` yet, which is the case for standalone network, this means that quickstart has to wait at least 5 mins for the 64th ledger to be emitted and first archive published, before soroban rpc will be ready.